package.json

```diff
@@ -7,39 +7,44 @@
   "author": "jbcpollak <josh@6river.com> (http://6river.com/)",
   "main": "server/server.js",
   "engines": {
-    "node": "5.7.1",
+    "node": "6.2.1",
     "npm": "3.7.2"
```

Comment:
I'd remove this line - we don't really care about npm version now that 3.x is standard.